src/api.ts

```diff
@@ -75,10 +102,10 @@ async function getVisualizationFromBackend(
   return {} as PopulatedVisualization;
 }
 
-export async function getVisualisation(): Promise<PopulatedVisualization> {
```

Comment:
is the async needed?

Reply:
no, there is no `await` in this function; it just returns a promise (for when `source` is not `local`)

Reply:
lol I reviewed the wrong line, thought you added it 😅

Reply:
no problem 😂